lldb/source/Symbol/SaveCoreOptions.cpp

```diff
@@ -145,6 +145,28 @@ SaveCoreOptions::GetThreadsToSave() const {
   return thread_collection;
 }
 
+uint64_t SaveCoreOptions::GetCurrentSizeInBytes(Status &error) {
```

Comment:
Suggestion: `llvm::Expected<uint64_t>`. Callers will be forced to handle the error instead of dropping it.